dashboard/src/components/widget/select/che-select.directive.ts

```diff
@@ -16,34 +16,76 @@
  * @author Oleksii Orel
  */
 export class CheSelect {
+  restrict: string = 'E';
+  replace: boolean = true;
+  transclude: boolean = true;
+
+  scope: {
+    [propName: string]: string
+  };
 
   /**
    * Default constructor that is using resource
    * @ngInject for Dependency injection
    */
   constructor() {
-    this.restrict = 'E';
-
-    this.replace = true;
-    this.transclude = true;
-    this.templateUrl = 'components/widget/select/che-select.html';
```

Comment:
This approach allows me to explicitly associate element name and ng-messages directive to properly show a validation message.
Additional pros is it will work faster as we avoid extra request.